Makefile

```
@@ -0,0 +1,35 @@
+CDH_COMMIT ?=
+
+default_cdh_commit := ed809e5
+
+ifeq ($(CDH_COMMIT),)
+    cdh_commit := $(default_cdh_commit)
+else
+    cdh_commit := $(CDH_COMMIT)
+endif
```

Could we directly use
```suggestion
CDH_COMMIT ?= ed809e5
```

They seem to be equal